pkgs/applications/audio/vmpk/default.nix

```diff
@@ -1,19 +1,28 @@
-{ stdenv, fetchurl, cmake, pkgconfig, alsaLib, libjack2, qt4 }:
+{ mkDerivation, lib, fetchurl, cmake, pkg-config, alsaLib, libjack2
+, qttools, qtx11extras, drumstick
+, libxcb, libXdmcp, libpthreadstubs
+, docbook-xsl-nons, docbook_xml_dtd_44
+}:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
   pname = "vmpk";
-  version = "0.5.1";
+  version = "0.7.2";
 
   src = fetchurl {
     url = "mirror://sourceforge/${pname}/${version}/${pname}-${version}.tar.bz2";
-    sha256 = "11fqnxgs9hr9255d93n7lazxzjwn8jpmn23nywdksh0pb1ffvfrc";
+    sha256 = "5oLrjQADg59Mxpb0CNLQAE574IOSYLDLJNaQ/9q2cMQ=";
   };
 
-  nativeBuildInputs = [ cmake pkgconfig ];
+  nativeBuildInputs = [ cmake pkg-config qttools ];
 
-  buildInputs = [ alsaLib libjack2 qt4 ];
+  buildInputs = [
```

Comment:
Please ignore CMake/pkg-config warnings and do not add static dependencies to buildInputs. ([Issue report](https://gitlab.kitware.com/cmake/cmake/-/issues/18158).) This seems OK:

    nativeBuildInputs = [ cmake docbook-xsl-nons pkg-config qttools ];
    buildInputs = [ drumstick qtx11extras ];